docs/templates/jenkins/scan-repository.jenkinsfile

```diff
@@ -50,6 +50,11 @@ pipeline {
         // The 'frogbot' executable and other tools it needs will be downloaded through this repository.
         // JF_RELEASES_REPO= ""
 
+        // [Optional]
+        // Avoid printing additional output in pull request comments
+        // For example, a JFrog Advanced Security message in pull request comments.
+        // JF_AVOID_EXTRA_MESSAGES= "true"
```

Comment:
Norice that for other variables in the table, we use the value "TRUE" and not "true". Let's maintain consistency.